projects/laji/src/styles/laji.scss

```diff
@@ -1360,3 +1360,41 @@ span.simple-option {
     min-width: 250px !important;
   }
 }
+
+.map-legend {
```

Comment:
Could this be moved to a new file? Eg. `src/styles/map/ykj-map.scss`. Also the map-legend name is maybe a bit too general for global namespace so either it should be renamed, or imported only where it's needed.

Reply:
I renamed the class as map-color-legend and moved it to its own file